### DISCUSSION TITLE
Black bars issue

### DISCUSSION THREAD

**Discussion by chesterflaps** [awawa-dev/HyperHDR#429] [General]
I have the black bar option selected as some shows have small black bars top and bottom, some have larger and some have none. The issue I am having is more at the sides of the screen when watching anything with dark scenes. It's as if the black bar detection thinks there are huge black bars at the side of the screen and then matches the colour which can be anything up to 12 inches in from the side of the screen. Is there a way to stop this? This does happen at the top and bottom too which is also annoying as it will suddenly switch some lights on when there's something dark at the top or bottom of the screen.

Thank you.

**Comment by awawa-dev** [comment]
Hi
If you can pause your video source on problematic frame, please provide two screenshots from HyperHDR live preview window to show the problem (with and without blackbar detector enabled). Also your HyperHDR blackbar configuration is needed.
